app/metrics.yaml

```diff
@@ -7464,6 +7464,22 @@ credit_cards:
     metadata:
       tags:
         - Autofill
+  save_prompt_shown:
+    type: event
+    description: |
+      The autofill save prompt is shown.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/26089
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/26095
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: 105
```

Comment:
105 would mean this expires next release.

Reply:
Changed that to 118, based on the telemetry expiry date renewal in https://github.com/mozilla-mobile/fenix/issues/26121.